### ISSUE TITLE
Access violation moving Non Visual Componentes

### ISSUE THREAD

**Issue by Blueicaro** [castle-engine/castle-engine#528]
Hello, I have a project, with a Label, this label has as children a Non Visual Component, and this has as child a BitMapFont.
I got a video, that shows the problem. 
https://youtu.be/YSmaJODdQ2U

/BlueIcaro

**Comment by michaliskambi** [comment]
Thank you for the report, and the video clearly showing the issue. 

As it happens, this bug is already fixed in latest CGE :) This was fixed in https://github.com/castle-engine/castle-engine/commit/ecd3280972626cfc6008aef929e0407d3a9dff74 , around 2 weeks ago, in response to another bugreport https://github.com/castle-engine/castle-engine/issues/520 . (while that bugreport shows a bit different situation, the end cause was the same, and the fix also helps in this case -- I just tested.) The _"Non-Visual Components"_ tree item cannot be dragged now.

Looking at your video, you use CGE from March 25th ( because your log shows the engine version `024802c94f356b7658ef98672e81d39304cb60dd`, this commit, https://github.com/castle-engine/castle-engine/commit/024802c94f356b7658ef98672e81d39304cb60dd ).

So if you just upgrade to latest CGE from https://castle-engine.io/download , the problem will be fixed.
